ui/src/views/offering/AddComputeOffering.vue

```diff
@@ -764,6 +764,7 @@ export default {
         this.fetchStorageTagData()
         this.fetchDeploymentPlannerData()
       } else if (this.isDomainAdmin()) {
+        this.form.ispublic = false
```

Comment:
LGTM. Minor suggestion: @nvazquez would it be better to put this inside `created()` itself

Reply:
Thanks @shwstppr, I've applied your suggestion